tests/test_redis_storage.py

```diff
@@ -4,7 +4,7 @@
 import json
 import time
 import uuid
-from typing import Any, Callable, Dict, MutableMapping, Optional, cast
+from typing import Any, Callable, MutableMapping, cast
```

Comment:
Please revert these. Only reason they don't cause a test failure is the `from __future__` import above. We'll automatically update all these annotations once Python 3.8 is unsupported.